src/data/balance/pokemon-evolutions/gen2-pokemon-family-evolutions.ts

```diff
@@ -0,0 +1,216 @@
+      AMBIPOM_EVO_LEVEL,
+    ),
+  ],
+  [Species.SUNKERN]: [new SpeciesEvolution(Species.SUNFLORA, 1, EvolutionItem.SUN_STONE, null, GENERIC_ITEM_EVO_LEVEL)],
+  [Species.YANMA]: [
+    new SpeciesEvolution(
+      Species.YANMEGA,
+      1,
+      null,
+      new SpeciesEvolutionCondition((p) => p.moveset.filter((m) => m.moveId === MoveId.ANCIENT_POWER).length > 0),
+      YANMEGA_EVO_LEVEL,
+    ),
+  ],
+  [Species.WOOPER]: [new SpeciesEvolution(Species.QUAGSIRE, 20, null, null)],
+  [Species.MURKROW]: [
+    new SpeciesEvolution(Species.HONCHKROW, 1, EvolutionItem.DUSK_STONE, null, GENERIC_ITEM_EVO_LEVEL),
+  ],
+  [Species.MISDREAVUS]: [
+    new SpeciesEvolution(Species.MISMAGIUS, 1, EvolutionItem.DUSK_STONE, null, GENERIC_ITEM_EVO_LEVEL),
+  ],
+  /** Wynaut is from Gen 3 */
+  [Species.WYNAUT]: [new SpeciesEvolution(Species.WOBBUFFET, 15, null, null)],
+  [Species.GIRAFARIG]: [
+    new SpeciesEvolution(
+      Species.FARIGIRAF,
+      1,
+      null,
+      new SpeciesEvolutionCondition((p) => p.moveset.filter((m) => m.moveId === MoveId.TWIN_BEAM).length > 0),
+      FARIGARIF_EVO_LEVEL,
+    ),
+  ],
+  [Species.PINECO]: [new SpeciesEvolution(Species.FORRETRESS, 31, null, null)],
+  [Species.DUNSPARCE]: [
+    new SpeciesFormEvolution(
+      Species.DUDUNSPARCE,
+      "",
+      "three-segment",
+      1,
+      null,
+      new SpeciesEvolutionCondition((p) => {
+        let ret = false;
+        if (p.moveset.filter((m) => m.moveId === MoveId.HYPER_DRILL).length > 0) {
+          globalScene.executeWithSeedOffset(() => (ret = !randSeedInt(4)), p.id);
+        }
+        return ret;
+      }),
+      DUDUNSPARCE_EVO_LEVEL,
+    ),
+    new SpeciesEvolution(
+      Species.DUDUNSPARCE,
+      1,
+      null,
+      new SpeciesEvolutionCondition((p) => p.moveset.filter((m) => m.moveId === MoveId.HYPER_DRILL).length > 0),
+      DUDUNSPARCE_EVO_LEVEL,
+    ),
+  ],
+  [Species.GLIGAR]: [
+    new SpeciesEvolution(
+      Species.GLISCOR,
+      1,
+      EvolutionItem.RAZOR_FANG,
+      new SpeciesEvolutionCondition((_p) => globalScene.arena.isTimeOfDay([TimeOfDay.NIGHT, TimeOfDay.DUSK])),
+      GENERIC_ITEM_EVO_LEVEL,
+    ),
+  ],
+  [Species.SNUBBULL]: [new SpeciesEvolution(Species.GRANBULL, 23, null, null)],
+  [Species.SNEASEL]: [
+    new SpeciesEvolution(
+      Species.WEAVILE,
+      1,
+      EvolutionItem.RAZOR_CLAW,
+      new SpeciesEvolutionCondition((_p) => globalScene.arena.isTimeOfDay([TimeOfDay.NIGHT, TimeOfDay.DUSK])),
+      GENERIC_ITEM_EVO_LEVEL,
+    ),
+  ],
+  [Species.TEDDIURSA]: [new SpeciesEvolution(Species.URSARING, 30, null, null)],
+  [Species.URSARING]: [
+    new SpeciesEvolution(
+      Species.URSALUNA,
+      1,
+      EvolutionItem.PEAT_BLOCK,
+      new SpeciesEvolutionCondition((_p) => globalScene.arena.isTimeOfDay([TimeOfDay.NIGHT, TimeOfDay.DUSK])),
+      ADVANCED_ITEM_EVO_LEVEL,
+    ), //Ursaring does not evolve into Bloodmoon Ursaluna
```

Comment:
```suggestion
    ), // Ursaring does not evolve into Bloodmoon Ursaluna
```
Not sure if this comment is necessary in the first place, but eh

Reply:
This was here originally ¯\_(ツ)_/¯ might be helpful to people who aren't as familiar with gen 8/9